packages/auth/src/model/public_types.ts

```diff
@@ -291,6 +291,12 @@ export interface Auth {
     error?: ErrorFn,
     completed?: CompleteFn
   ): Unsubscribe;
+  /**
```

Comment:
If this is publicly visible, please apply fixes suggested for identical text above, thanks!